openapi_python_client/templates/endpoint_macros.py.jinja

```diff
@@ -11,6 +11,20 @@ if {{ parameter.python_name }} is not UNSET:
 {% endif %}
 {% endmacro %}
 
+{% macro cookie_params(endpoint) %}
+{% if endpoint.cookie_parameters %}
+    {% for parameter in endpoint.cookie_parameters %}
+        {% if parameter.required %}
+cookies["{{ parameter.python_name | kebabcase}}"] = {{ parameter.python_name }}
+        {% else %}
+if {{ parameter.python_name }} is not UNSET:
+    cookies["{{ parameter.python_name | kebabcase}}"] = {{ parameter.python_name }}
```

Comment:
I think that cookies are both case-sensitive and don't follow the same kebab-case convention as headers. So we probably want this to be:

```suggestion
cookies["{{ parameter.name}}"] = {{ parameter.python_name }}
        {% else %}
if {{ parameter.python_name }} is not UNSET:
    cookies["{{ parameter.name}}"] = {{ parameter.python_name }}
```

Reply:
Yes, that's right. 
I will update the test case and commit it again